sklearn/metrics/cluster/unsupervised.py

```diff
@@ -288,6 +289,13 @@ def calinski_harabaz_score(X, labels):
             (intra_disp * (n_labels - 1.)))
 
 
+@deprecated("Function 'calinski_harabaz_score' was renamed to "
+            "'calinski_harabasz_score' "
+            "in version 0.21 and will be removed in release 0.23. ")
```

Comment:
redundant space at the end. release 0.23 -> version 0.23 (you need to update the test accordingly)